src/models/parallelCoordinates.js

```diff
@@ -366,7 +377,7 @@ nv.models.parallelCoordinates = function() {
             }
 
             function dragMove(d) {
-                dragging[d.key] = Math.min(availableWidth, Math.max(0, this.parentNode.__origin__ += d3.event.x));
+                dragging[d.key] = Math.min(width, Math.max(0, this.parentNode.__origin__ += d3.event.x));
```

Comment:
On my case it doesn't work with availableWidth and works well with width but your comment make sense. I will try to understand why so I removed this from the PR for now